packages/blockchain-link/src/workers/solana/index.ts

```diff
@@ -103,32 +129,49 @@ const pushTransaction = async (request: Request<MessageTypes.PushTransaction>) =
     const rawTx = request.payload.startsWith('0x') ? request.payload.slice(2) : request.payload;
     const api = await request.connect();
 
-    const { lastValidBlockHeight } = await api.getLatestBlockhash('finalized');
+    const {
+        value: { lastValidBlockHeight },
+    } = await api.rpc.getLatestBlockhash({ commitment: 'finalized' }).send();
 
-    const txBuffer = Buffer.from(rawTx, 'hex');
+    const txByteArray = getBase16Encoder().encode(rawTx);
+    const transaction = getTransactionDecoder().decode(txByteArray);
+    assertTransactionIsFullySigned(transaction);
 
-    try {
-        await api.sendRawTransaction(txBuffer, {
-            skipPreflight: true,
-            maxRetries: 0,
-        });
+    const compiledMessage = getCompiledTransactionMessageDecoder().decode(transaction.messageBytes);
+    const message = decompileTransactionMessage(compiledMessage);
+    if (isDurableNonceTransaction(message)) {
+        // TODO: Handle durable nonce transactions.
+        throw new Error('Unimplemented: Confirming durable nonce transactions');
+    }
```

Comment:
If you ever want to support these, there is a version of `sendAndConfirmTransactionFactory` that can confirm durable nonce transactions.